src/Authentication/Program.cs

```diff
@@ -416,19 +423,21 @@ void ConfigurePostgreSql()
 {
     if (builder.Configuration.GetValue<bool>("PostgreSqlSettings:EnableDBConnection"))
     {
-        ConsoleTraceService traceService = new ConsoleTraceService { IsDebugEnabled = true };
-
-        string connectionString = string.Format(
-            builder.Configuration.GetValue<string>("PostgreSqlSettings:AdminConnectionString"),
-            builder.Configuration.GetValue<string>("PostgreSqlSettings:AuthenticationDbAdminPassword"));
+        //Yuniql Migration Scripts
+        ConsoleTraceService traceService = new() { IsDebugEnabled = true };
 
         string workspacePath = Path.Combine(Environment.CurrentDirectory, builder.Configuration.GetValue<string>("PostgreSqlSettings:WorkspacePath"));
         if (builder.Environment.IsDevelopment())
         {
+            string connectionString = string.Format(
```

Comment:
Just wondering that this is under if development check, do we have similar for other environments?

Reply:
no, it's only for testing scripts against a local db during dev

Reply:
we will simplify this in a later PR